### ISSUE TITLE
after 1 epoch,  you  don't shuffle?

### ISSUE THREAD

**Issue by Wallace-he** [jmiller656/EDSR-Tensorflow#46]

**Comment by kong1060553768** [comment]
I also see this problem, and'train_set'seems to remain unchanged.

**Comment by kong1060553768** [comment]
@Wallace-he   Have you solved this problem?

**Comment by jmiller656** [comment]
This is correct. I'm actually working on migrating this over to keras (for multiple reasons), this will be one of the problems solved by that. Could use some help figuring out a problem with model saving in keras tho

**Comment by kong1060553768** [comment]
I didn't understand it before. But I want to ask you how I want to change the PSNR value after each test.

**Comment by kong1060553768** [comment]
Sorry, this is my mistake! Now I want to output the PSNR value in each test. How can I achieve this?

**Comment by jmiller656** [comment]
You can watch this using tensorboard, that's how I recommend doing it.

If you want to print PSNR, you'll have to edit the train loop to evaluate PSNR from the graph and print that

**Comment by kong1060553768** [comment]
Thank you. I'm sorry to bother you many times. But with your checkpoints and your own training results, the PSNR value of reconstructed images on set5 and test set and individual other pictures is even lower than that of traditional methods. Why?
